xcodeproj/internal/bazel_integration_files/create_lldbinit.sh

```diff
@@ -82,6 +82,8 @@ touch "$lldbinit"
 
 readonly required_source='command source ~/.lldbinit-rules_xcodeproj'
 if ! grep -m 1 -q "$required_source" "$lldbinit"; then
```

Comment:
I think this will do it?
```suggestion
if ! grep -m 1 -q "^$required_source" "$lldbinit"; then
```

Reply:
That doesn't seem to work, it adds to the same line due to `echo` below:


```sh
cat ~/.lldbinit
command echo "hello world"%       

cat ~/.lldbinit                                                                                                 
command echo "hello world"command source ~/.lldbinit-rules_xcodeproj
```

Reply:
Was missing the `$` at end.

This works: `"^$required_source$"`